packages/wells/src/client/api/wellsApi.ts

```diff
@@ -37,6 +38,33 @@ export class WellsAPI {
     return baseUrl
   }
 
+  private convertToWkt(filter: WellFilter): WellFilterAPI {
+    filter.polygon!.wktGeometry = convertGeoJsonToWKT(filter.polygon!.geoJsonGeometry!)
+    return this.convertToApiWellFilter(filter)
+  }
+
+  private convertToApiWellFilter(filter: WellFilter): WellFilterAPI {
+    let polygon: PolygonFilterAPI | undefined;
+    if (filter.polygon) {
+      polygon = {
+        geometry: filter.polygon.wktGeometry!, crs: filter.polygon.crs, geometryType: "wkt"
+      }
+    } else {
+      polygon = undefined;
+    }
```

Comment:
nit: I prefer this, but it's up to you :) 
```suggestion
    let polygon: PolygonFilterAPI | undefined = undefined;
    if (filter.polygon) {
      polygon = {
        geometry: filter.polygon.wktGeometry!, crs: filter.polygon.crs, geometryType: "wkt"
    }
```